src/assets/theme.css

```diff
@@ -16,6 +16,7 @@
   --fg-color: #fff;
   --button-bg: #333;
   --button-hover-bg: #444;
+  --list-hover-color: #bcbcbc;
```

Comment:
Just need to make sure this is not too dark. Although, it’s not list-hover-color anymore. Let’s call it `dim`.